Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -170,8 +203,22 @@
             "label": "Runtime Stack",
             "defaultValue": "",
             "required": true,
-            "properties": {
```

Comment:
This change shouldn't be reverted.

Reply:
changes reverted back to original. It got messed up during rebasing the code.